src/sub-blocks/HubspotForm/HubspotForm.scss

```diff
@@ -110,6 +110,16 @@ $block: '.#{$ns}hubspot-form';
                     border: 1px solid var(--g-color-line-generic-active);
                 }
             }
+
+            .hs-fieldtype-intl-phone.hs-input .hs-input {
+                float: none;
+                /* stylelint-disable-next-line declaration-no-important */
+                width: 100% !important;
```

Comment:
hubspot script set important for these elements, so i need to also use important

Reply:
Could you please add this comment into the file?

Reply:
Done